docs/UserGuide.md

```diff
@@ -1147,7 +1166,7 @@ copy), delete the `werkItResources` directory and restart WerkIt! to recreate th
 ### 1. My terminal in Windows is not displaying any colours in the application.
 
 **A**: Please install Microsoft's newer terminal 'Windows Terminal' from the Microsoft Store via this
-[link](https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701).
+[link][1].
```

Comment:
I think this question is great but first time users might not know that this app has colours. Hence might change to a more suitable question like "My terminal is showing some weird symbols [paste a screenshot of the fail colour render]"

Reply:
Thank you for your feedback. It has been implemented in Commit 692f60b